### ISSUE TITLE
Training with hyperparams provided in best weight, but got different results.

### ISSUE THREAD

**Issue by yen52205** [Dohppak/EMOPIA_cls#5]
Hi, 
I tried the midi_cls/train_test.py on environment below:

pytorch == 1.8.0
python == 3.8.5
CUDA ==10.1

I trained the models with A/V/AV best hyperparameters that provided in best weights folder.
But got different results from the json file provided in best weight file.

Below is the hyperparams and results log produced by original train_test.py code. (I transformed them to .txt for uploading.)
[arva_hparams.txt](https://github.com/SeungHeonDoh/EMOPIA_cls/files/7443400/arva_hparams.txt)
[arva_results_last.txt](https://github.com/SeungHeonDoh/EMOPIA_cls/files/7443401/arva_results_last.txt)
[av_hparams.txt](https://github.com/SeungHeonDoh/EMOPIA_cls/files/7443402/av_hparams.txt)
[av_results_last.txt](https://github.com/SeungHeonDoh/EMOPIA_cls/files/7443403/av_results_last.txt)

* I already added global seed setting in the script.
* 
![image](https://user-images.githubusercontent.com/48942210/139467764-d42e82d1-b6b3-432f-8842-574d6e0fd599.png)

I'm really confused about the reason, is there any way to make sure what's going wrong?
thanks.
